src/app/zap-templates/zcl/data-model/chip/access-control-cluster.xml

```diff
@@ -60,13 +68,39 @@ limitations under the License.
     <item fieldId="1" name="Data" type="octet_string" length="128" isFabricSensitive="true"/>
   </struct>
 
+  <struct name="AccessRestrictionStruct">
+    <cluster code="0x001F"/>
+    <item fieldId="0" name="Type" type="AccessRestrictionTypeEnum" min="0x00" max="0x03"/>
+    <item fieldId="1" name="Id" type="int32u" isNullable="true"/>
+  </struct>
+
+  <struct name="CommissioningAccessRestrictionEntryStruct">
```

Comment:
For future reference: having the order in the XML match the spec order makes review a _lot_ simpler, because then one can just linearly go through both instead of trying to match up N things to N other things in a different order.... (N vs N^2 time).

Reply:
Done and noted.